test/parallel/test-dns-regress-7070.js

```diff
@@ -4,5 +4,5 @@ const assert = require('assert');
 const dns = require('dns');
 
 // Should not raise assertion error. Issue #7070
-assert.throws(() => dns.resolveNs([])); // bad name
-assert.throws(() => dns.resolveNs('')); // bad callback
+assert.throws(() => dns.resolveNs([]), /"name" argument must be a string/); // bad name
+assert.throws(() => dns.resolveNs(''), /"callback" argument must be a function/); // bad callback
```

Comment:
If you're checking the entire error message, you can add `^` and `$` to the regular expression as well.